azurecompute/pom.xml

```diff
@@ -56,6 +56,11 @@
       <version>${project.parent.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.apache.jclouds.driver</groupId>
+      <artifactId>jclouds-sshj</artifactId>
+      <version>${project.parent.version}</version>
```

Comment:
We shouldn't force users to a concrete SSH driver. Wouldn't be enough to just add the BouncyCastle driver dependency? Can we get rid of that dependency, anyway?